app/controllers/cold_messages_controller.rb

```diff
@@ -51,6 +52,14 @@ def require_signed_hiring_agreement!
     end
   end
 
+  def update_developer_response_rate
+    return unless response.redirect?
+
+    UpdateDeveloperResponseRateJob
+      .set(wait: Rails.application.config.developer_response_grace_period)
+      .perform_later(developer)
+  end
+
```

Comment:
This feels like a round about way to queue the job. Let's instead add this to `Messages::Notifications` or inline to the controller.

Reply:
I didn't like it here either, but it didn't seem to fit in with the Notifications either. It possibly deserves it's own class?
I've moved it in with the Notifications for now, as suggested.